packages/config/lib/options.ts

```diff
@@ -518,6 +522,11 @@ export const breakingOptions: Array<BreakingOption> = [
     name: 'experimentalShadowDomSupport',
     errorKey: 'EXPERIMENTAL_SHADOW_DOM_REMOVED',
     isWarning: true,
+  }, {
+    name: 'experimentalStudio',
+    errorKey: 'EXPERIMENTAL_STUDIO_REMOVED',
+    isWarning: true,
+    showInLaunchpad: true,
```

Comment:
Why is this needed 2x?

Reply:
This option needs to be removed in the options above: https://github.com/cypress-io/cypress/pull/20515/files#diff-3327662b7aa1effb2d97569ca234bf56dbd83bc0664c63c9be165e1923615e65R178

We also need to remove this config option from the types and json schema in cypress/cli

Reply:
<img width="955" alt="Screenshot 2022-03-09 at 11 28 58 AM" src="https://user-images.githubusercontent.com/27690333/157485727-e2e5acb3-e4d7-45c7-87ec-ba1512380d5a.png">

If I remove it from the breakingOptions, then it won't display a warning in the terminal. The breaking options are what we use in the validation function

Reply:
The breakingRootOptions allows the warning to show up in open mode as well